static/js/unread.js

```diff
@@ -413,6 +413,22 @@ exports.update_unread_topics = function (msg, event) {
     );
 };
 
+exports.update_unread_moved_topic = function (msg) {
```

Comment:
Does this need to be different than `exports.update_unread_topics`?  It seems like we could just change `new_topic` to `msg.topic` in the previous function and not need to add a new function, unless I'm missing something.

Reply:
`new_topic` is undefined for moving topics, so I am not sure how to validate that on this case.